Source/Urho3D/AngelScript/ScriptFile.h

```diff
@@ -81,14 +81,24 @@ class URHO3D_API ScriptFile : public Resource, public ScriptEventListener
 
     /// Query for a function by declaration and execute if found.
     bool Execute(const String& declaration, const VariantVector& parameters = Variant::emptyVariantVector, bool unprepare = true);
+    /// Query for a function by declaration and execute and return result if found.

```

Comment:
Is it possible to inspect AS context to find out what is the type of the return value? Passing the return type on each call here seems not that user-friendly.

Reply:
Ok It seems possible to get return value using asIScriptFunction::GetReturnTypeId()